english/warriors-game/watch-warriors.md

```diff
@@ -13,4 +13,4 @@ good old days, when the deejay plays a Tevin Campbell track at your school dance
 that no girl can resist a slow dance. You ponder about what else was great about the 90's.
 
 Michael Jordan was pretty great. LeBron James is great too. But not that great. They would be 
-great if they had not traded Wiggins. That was a dumb move. Warriors win in 5.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.